src/main/resources/templates/resource2.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+		<title>National Cybersecurity Laboratories - Resource</title>
+		<object th:include="fragments/header :: header" th:remove="tag" />
+	</head>
+	<body class="mybg loginbackground">
+	<div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
+	<div class="container">
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<!--here make changes-->
+		<!--<div class="tab-content col-sm-10 dashpanel_admin">-->
+		<div class="row dashpanel col-sm-12">
+		<!--<div class="tab-content col-sm-10">-->
+		<h2 class="team-heading team-heading-current-team" th:text="#{resource.heading.current}">text heading from .properties</h2>
+
+		<ul class="topnav col-sm-2">
+			<li>
+				<a href="#">Resource Type</a>
+				<ul>
+					<li class="active"><a data-toggle="tab" href="#vul">Vulnerability Environments</a></li>
+					<li><a data-toggle="tab" href="#bigdata">Big-data Platforms</a></li>
+					<li><a data-toggle="tab" href="#blockchain">Blockchain Platforms</a></li>
+					<li><a data-toggle="tab" href="#mobile">Mobile Emulators</a></li>
+						<!--<ul>-->
+							<!--<li><a href="#">/a></li>-->
+							<!--<li><a href="#"></a></li>-->
+							<!--<li><a href="#"></a></li>-->
+						<!--</ul>-->
+				</ul>
+			</li>
+			<li><a href="#">Network Type</a>
+				<ul>
+					<li><a href="#">Single Host</a></li>
+					<li><a href="#">Client-Server</a></li>
+					<li><a href="#">P2P network</a></li>
+					<li><a href="#">Distributed Computing</a></li>
+					<li><a href="#">Mobile</a></li>
+				</ul>
+			</li>
+			<li><a href="#">Year Created</a>
+				<ul>
+					<li><a href="#">2016</a></li>
+					<li><a href="#">2015</a></li>
+					<li><a href="#">2014</a></li>
+
+				</ul>
+			</li>
+
+		</ul>
+
+		<div class="col-sm-9">
+			<div id="vul" class="tab-panel fade in active">
+				<h3>System Vulnerability Environments</h3>
+
+					<!--<div class="form-inline left">-->
+							<!--&lt;!&ndash;<input type="search" class="filterTable" placeholder="Search..."/>&ndash;&gt;-->
+							<!--<button type="button" class="form-control"><i class="fa fa-search"></i></button>-->
+					<!--</div>-->
+
+
+				<!--<div class="input-group">-->
+					<!--<input type="search" class="form-control" placeholder="Search...">-->
+					<!--<span class="input-group-btn">-->
+    				<!--<button class="btn btn-default" button type="button">Go!</button>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)